error_reporting/setup.py

```diff
@@ -52,13 +52,13 @@
 
 REQUIREMENTS = [
     'google-cloud-core >= 0.25.0, < 0.26dev',
-    'google-cloud-logging >= 1.0.0, < 2.0dev',
+    'google-cloud-logging >= 1.1.0, < 1.2dev',
```

Comment:
I'm not positive that tightening the upper bound is the Right Thing(TM).  ISTM that semver requires that we do not make any breaking changes to `google-cloud-logging` before a 2.0.0 release.